submissions/Nemooochka/js-dom/js/script.js

```diff
@@ -0,0 +1,155 @@
+        'director': 'Peter Farrelly',
+        'writers': 'Nick Vallelonga, Brian Hayes Currie (as Brian Currie)',
+        'stars': 'Viggo Mortensen, Mahershala Ali, Linda Cardellini '
+    },
+    {
+        'year': 'y_2017',
+        'name': 'The Shape of Water',
+        'poster': 'https://m.media-amazon.com/images/M/MV5BNGNiNWQ5M2MtNGI0OC00MDA2LWI5NzEtMmZiYjVjMDEyOWYzXkEyXkFqcGdeQXVyMjM4NTM5NDY@._V1_UX182_CR0,0,182,268_AL_.jpg',
+        'description': 'At a top secret research facility in the 1960s, a lonely janitor forms a unique relationship with an amphibious creature that is being held in captivity.',
+        'director': 'Guillermo del Toro',
+        'writers': 'Guillermo del Toro (screenplay by), Vanessa Taylor (screenplay by) |',
+        'stars': 'Sally Hawkins, Octavia Spencer, Michael Shannon'
+    },
+    {
+        'year': 'y_2016',
+        'name': 'Moonlight',
+        'poster': 'https://m.media-amazon.com/images/M/MV5BNzQxNTIyODAxMV5BMl5BanBnXkFtZTgwNzQyMDA3OTE@._V1_UX182_CR0,0,182,268_AL_.jpg',
+        'description': 'A young African-American man grapples with his identity and sexuality while experiencing the everyday struggles of childhood, adolescence, and burgeoning adulthood.',
+        'director': 'Barry Jenkins',
+        'writers': 'Barry Jenkins (screenplay by), Tarell Alvin McCraney (story by)',
+        'stars': 'Mahershala Ali, Naomie Harris, Trevante Rhodes '
+    },
+    {
+        'year': 'y_2015',
+        'name': 'Spotlight',
+        'poster': 'https://m.media-amazon.com/images/M/MV5BMjIyOTM5OTIzNV5BMl5BanBnXkFtZTgwMDkzODE2NjE@._V1_UX182_CR0,0,182,268_AL_.jpg',
+        'description': 'The true story of how the Boston Globe uncovered the massive scandal of child molestation and cover-up within the local Catholic Archdiocese, shaking the entire Catholic Church to its core.',
+        'director': 'Tom McCarthy',
+        'writers': 'Josh Singer, Tom McCarthy',
+        'stars': 'Mark Ruffalo, Michael Keaton, Rachel McAdams'
+    },
+    {
+        'year': 'y_2014',
+        'name': 'Birdman',
+        'poster': 'https://m.media-amazon.com/images/M/MV5BODAzNDMxMzAxOV5BMl5BanBnXkFtZTgwMDMxMjA4MjE@._V1_UX182_CR0,0,182,268_AL_.jpg',
+        'description': 'A washed-up superhero actor attempts to revive his fading career by writing, directing, and starring in a Broadway production.',
+        'director': 'Alejandro G. Iñárritu',
+        'writers': 'Alejandro G. Iñárritu, Nicolás Giacobone',
+        'stars': 'Michael Keaton, Zach Galifianakis, Edward Norton'
+    }
+];
+
+const initialMovie = data[0];
+const navLink = document.querySelectorAll('.sidebar a');
+const sidebar = document.querySelector(".sidebar");
+const menuChevron = document.querySelector(".menu-chevron");
+
+const toggleSidebar = () => {
+    sidebar.classList.toggle('open');
+};
+
+/* get a year (last part of url) from href */
+const getLastItem = thePath => thePath.substring(thePath.lastIndexOf('/') + 1);
+
+function generateView(obj) {
```

Comment:
I'd recommend to use `obj` destructuring here.
```suggestion
function generateView({poster, name, description, director, writer, stars}) {
```